.github/workflows/windows.yml

```diff
@@ -428,7 +428,7 @@ jobs:
           PATH="$(cygpath "${USERPROFILE}")/my-cache/${{ matrix.dir }}/bin:/c/msys64/usr/bin:$PATH"
           [ '${{ matrix.type }}' = 'Debug' ] && options+=' -DCMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG='
           [ '${{ matrix.type }}' = 'Release' ] && options+=' -DCMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE='
-          cmake -B bld -G ${{ contains(matrix.url, '/winlibs_mingw/') && 'Ninja' || 'MSYS Makefiles' }} ${options} \
+          cmake -B bld -G ${{ contains(matrix.url, '/winlibs_mingw/') && 'Ninja' || "'MSYS Makefiles'" }} ${options} \
```

Comment:
```suggestion
          cmake -B bld -G ?${{ contains(matrix.url, '/winlibs_mingw/') && 'Ninja' || 'MSYS Makefiles' }}" ${options} \
```

Reply:
your way will probably work too

Reply:
This is nicer, pushed an update.

Best not to think about the man-years saved if those `-G` args were just continious strings with no spaces in them!

Reply:
at least you're dealing with quoting in bash, and not cmd.  Its quoting rules are really arcane